apis/rdc.json

```diff
@@ -0,0 +1,3036 @@
+{
```

Comment:
Is there something else we need to document for using the API of the RDC? 
The description for the API for the RDC explains some different usage of the keys / password and so on

Reply:
I currently don't see any differences , you need to supply a username and a password (like access key) and it works.